app/models/cart.rb

```diff
@@ -32,20 +32,28 @@ def get_items # rubocop:disable AccessorMethodName
   end
 
   # Adds equipment model id to items hash
-  def add_item(equipment_model)
+  def add_item(equipment_model, _quantity = nil)
```

Comment:
There's so much code duplication here, I feel like we could totally refactor the `add_item` and `remove_item` methods with calls to `set_cart_item` / `edit_cart_item`. Honestly, we might want to do this in the controller rather than here (I'll comment above).